src/templates/default_function/requirements.txt

```diff
@@ -1 +1,2 @@
 snowflake-snowpark-python
+toml # For local development
```

Comment:
I don't think this is the right fix. The right think would be to updated `snowflake-snowpark-python` to use latest `snowflke-python-connector` which requires toml.

Knowing this will take time I would suggest adding explicite `snowflake-connector-python>=3.1.0`.